src/main/java/com/example/sccshop/ProductResponse.java

```diff
@@ -0,0 +1,57 @@
+package com.example.sccshop;
+
+/**
+ * @author Eddú Meléndez
+ */
+public class ProductResponse {
```

Comment:
Maybe this should be `Product` or `ProductDTO`, cause the response is an entire list of them? wdyt?